papermill/execute.py

```diff
@@ -147,7 +147,8 @@ def execute_notebook(notebook,
                      kernel_name=None,
                      progress_bar=True,
                      log_output=False,
-                     start_timeout=60):
+                     start_timeout=60,
+                     report_mode=None):
```

Comment:
I'd do `report_mode=False` here too as it's a flag

Reply:
ok thanks!